Makefile

```diff
@@ -1093,7 +1093,10 @@ $(RCPTS)/package.rcpt: $(RCPTS)/$(pack-sys).rcpt $(RCPTS)/source_tarball.rcpt
 
 # Build and include the monitor (watch_ldconfig) utility
 # RPM distros use a systemd service and DEB distros use a cronjob.
-$(RCPTS)/monitor.rcpt: $(RCPTS)/toolchain.rcpt
+# Run it after the last ldconfig pass because it ldconfig stages regenerate
+# ldconfig.filelist and potentially deleting the information generated in this
+# stage.
```

Comment:
Ack.  Considering this is just a source code comment, I plan to push it without updating this PR because it would trigger 46 new builds.
I hope that's OK.